examples/swap.yml

```diff
@@ -40,3 +40,4 @@ services:
 trust:
   org:
     - linuxkit
+    - library
```

Comment:
It's weird because gogland shows an extra line in the UI :(  Thanks for noticing, I'll fix this manually now and tinker with the settings to figure out why it's doing that